py-scripts/lf_ftp.py

```diff
@@ -103,18 +132,41 @@
 class FtpTest(LFCliBase):
     def __init__(self, lfclient_host="localhost", lfclient_port=8080, sta_prefix="sta", start_id=0, num_sta=0, radio="",
                  dut_ssid=None, dut_security=None, dut_passwd=None, file_size=None, band=None, twog_radio=None,
+                 file_name=None,
+                 profile_name=None, group_name=None,
                  sixg_radio=None, fiveg_radio=None, upstream="eth1", _debug_on=False, _exit_on_error=False, _exit_on_fail=False, ap_name="",
                  direction=None, duration=None, traffic_duration=None, ssh_port=None, kpi_csv=None, kpi_results=None,
-                 lf_username="lanforge", lf_password="lanforge", clients_type="Virtual", real_client_list=[],
-                 working_resources_list=[], hw_list=[], windows_list=[], mac_list=[], linux_list=[], android_list=[],
-                 eid_list=[], mac_id_list=[], devices_available=[], mac_id1_list=[], user_list=[], input_devices_list=[],
-                 real_client_list1=[], uc_avg=[], url_data=[], channel_list=[], mode_list=[], cx_list=[], dowebgui=False, device_list=[], test_name=None, result_dir=None):
+                 lf_username="lanforge", lf_password="lanforge", clients_type="Virtual", dowebgui=False, device_list=[], test_name=None, result_dir=None,
```

Comment:
Reformatting the parameters and changing their types makes for a confusing diff. Please avoid this in the future.